src/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -74,64 +75,66 @@ function frule(
     return F, ∂F
 end
 
+function _lu_pullback(ΔF::Tangent, A, pivot, F)
```

Comment:
I guess we had to move this because else it wouldn't type infer?
Would be good to add a comment